test/functional/p2p_fingerprint.py

```diff
@@ -109,24 +98,23 @@ def run_test(self):
 
         # Longest chain is extended so stale is much older than chain tip
         self.nodes[0].setmocktime(0)
-        tip = self.nodes[0].generatetoaddress(1, self.nodes[0].get_deterministic_priv_key().address)[0]
+        self.nodes[0].generatetoaddress(1, self.nodes[0].get_deterministic_priv_key().address)[0]
```

Comment:
The array index selection at the end of the line (`[0]`) isn't needed anymore if return value isn't stored in a variable.

Reply:
Good find, thanks! Fixed.